src/core/catalog.js

```diff
@@ -1161,7 +1167,14 @@ class Catalog {
           }
         }
 
-        const kids = currentNode.get("Kids");
+        let kids;
+        try {
+          kids = currentNode.get("Kids");
```

Comment:
Can't the other `currentNode.get("Type")` at line 1183 fail too ?

Reply:
In theory it obviously could, even if we've *thus far* not seen such a document; anyway I've updated the patch as suggested.